### PR TITLE
Item cleanup: Item drops, rupees, signs

### PR DESCRIPTION
**Pull request by engineer124**
Just a small bit of cleanup around items, mainly focused on gameplay_keep assets. I also did a small bit of docs for rupees where applicable, and the sign assets seems to have embedded themselves inside the item assets, so I just pulled the names from OoT since they seem identical

### CODE REVIEW

File: src/code/z_en_item00.c [hunk 1]
```diff
@@ -435,15 +471,15 @@ void func_800A6A40(EnItem00* this, PlayState* play) {
     this->actor.world.pos = player->actor.world.pos;
 
     if (this->actor.params <= ITEM00_RUPEE_RED) {
-        this->actor.shape.rot.y = this->actor.shape.rot.y + 960;
+        this->actor.shape.rot.y += 960;
```
Comment (hensldm) [inline src/code/z_en_item00.c:474]:
```suggestion
        this->actor.shape.rot.y += 0x3C0;
```

**Review by hensldm — APPROVED** [review]
Just 1 minor thing

File: src/code/z_en_item00.c [hunk 2]
```diff
@@ -143,39 +153,47 @@ void EnItem00_Init(Actor* thisx, PlayState* play) {
             this->unk154 = 0.03f;
             shadowOffset = 320.0f;
             break;
+
         case ITEM00_MAGIC_LARGE:
             Actor_SetScale(&this->actor, 0.044999998f);
```
Comment (EllipticEllipsis) [inline src/code/z_en_item00.c:158]: I would guess these are all `4.5f * 0.01f`.

File: src/code/z_en_item00.c [hunk 3]
```diff
@@ -706,85 +771,81 @@ void EnItem00_Draw(Actor* thisx, PlayState* play) {
...
-    gameplay_keep_Tex_061FE0, // Blue rupee
-    gameplay_keep_Tex_062000, // Red rupee
-    gameplay_keep_Tex_062040, // Orange rupee
-    gameplay_keep_Tex_062020  // Purple rupee
-};
+static TexturePtr sRupeeTextures[] = { gRupeeGreenTex, gRupeeBlueTex, gRupeeRedTex, gRupeeOrangeTex, gRupeePurpleTex };
 
+/**
+ * Draw Function used for Rupee types of En_Item00.
+ */
```
Comment (EllipticEllipsis) [inline src/code/z_en_item00.c:778]: I don't think this really gives more information than the function name.

File: src/code/z_en_item00.c [hunk 4]
```diff
@@ -118,12 +125,14 @@ void EnItem00_Init(Actor* thisx, PlayState* play) {
                 sp30 = -1;
             }
             break;
+
         case ITEM00_RECOVERY_HEART:
-            this->actor.home.rot.z = randPlusMinusPoint5Scaled(65535.0f);
+            this->actor.home.rot.z = randPlusMinusPoint5Scaled(0xFFFF);
```
Comment (EllipticEllipsis) [inline src/code/z_en_item00.c:130]: There's another couple of these in this file, could you do those too?

**Review by AngheloAlf — APPROVED** [review]

**Review by EllipticEllipsis — APPROVED** [review]

**Pull request merged**